.github/workflows/build.yml

```
@@ -60,6 +88,7 @@
           token: ${{ secrets.GITHUB_TOKEN }}
           tag: ${{ env.VERSION }}
           allowUpdates: true
+          body: ${{ env.VERSION }}
         env:
           export PATH: $PATH:/usr/local/go/bin
```

actionlint (1.7.4)

environment variable name "export PATH" is invalid. '&', '=' and spaces should not be contained

(env-var)